projects/igniteui-angular/src/lib/grids/columns/column.component.ts

```diff
@@ -1402,7 +1402,7 @@ export class IgxColumnComponent implements AfterContentInit, OnDestroy, ColumnTy
             col = this.allChildren.filter(c => !c.columnGroup && !c.hidden)[0] as any;
         }
         if (this.columnLayoutChild) {
-            return this.parent.childrenVisibleIndexes.find(x => x.column === this).index;
+            return this.parent.childrenVisibleIndexes.find(x => x.column === this)?.index;
```

Comment:
Would it not be better if this returned `|| -1` if the find fails? Otherwise not really doing what the getter signature implies.
And the fail I assume is to do with the async population of children? Just trying to figure out if there's a bigger problem this would mask.

Reply:
Actually, strike the last, the actual Angular tests should have no issue with async children